csclient/params/params.go

```diff
@@ -221,6 +221,20 @@ type PermRequest struct {
 	Write []string
 }
 
+// HomePage holds the request/response of an id/meta/home-page PUT or GET request.
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#put-idmeta
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetahome-page
+type HomePage struct {
```

Comment:
I'm with @urosj on this one. PermRequest & PermResponse above makes it very clear what it going on. If we ever want to change one of them we can in a backwards compatible way. On a slightly meta level is there a reason this information shouldn't go in extra-info removing the need for this discussion at all?